monty_hall_experiment.ipynb

```diff
@@ -0,0 +1,103 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from Monty_Hall_Fonction import Monty_Hall\n",
+    "\n",
+    "print(\"Voici les resultats de l'experience de Monty Hall \\npour 10 000 parties jouées pour chaque stratégie.\\n\")\n",
+    "plot = plt.bar([1, 2], Monty_Hall(), \n",
```

Comment:
Here no need to create the `plot` variable, `plt.bar(...)` is enough (same in second notebook).